### DISCUSSION TITLE
Need a [Fancy] Icon

### DISCUSSION THREAD

**Discussion by jamaluddinrumi** [responsively-org/responsively-app#982] [Ideas]
would it be nice if responsively is having a fancy icon with ambiance like developer tools, like say vscode?

**Comment by manojVivek** [comment]
That would be nice. 😄 
But unfortunately, we don't have great design expertise to try anything right now. But welcome any contributions to that as well.

**Comment by manojVivek** [comment]
@jamaluddinrumi Is your suggestion similar to this one: https://github.com/responsively-org/responsively-app/issues/1000 ?

**Comment by jamaluddinrumi** [comment]
yah, something like that

i made custom icon for my installation like this 
![Screen Shot 2023-06-29 at 10 52 39 AM](https://github.com/responsively-org/responsively-app/assets/446780/d763bdb1-cb4d-4440-bf70-76207ffbb7f9)

it's so much understandable for me over the current icon 

everyone are welcome to download my custom icon here https://www.dropbox.com/s/nz3391xp5db3s54/responsively-icon-for-mac.icns?dl=1

**Reply by ParamBirje** [reply]
That is a great design!

**Comment by markus-kraemer** [comment]
Keep it simple!

![Icon](https://github.com/responsively-org/responsively-app/assets/12045730/dddc322b-d5bf-4029-8df2-4a3b0ac610e7)

**Reply by markus-kraemer** [reply]
![Extra Large@1x](https://github.com/responsively-org/responsively-app/assets/12045730/589d8285-535e-485b-8081-3b3c3ff72884)

**Reply by markus-kraemer** [reply]
![lighter](https://github.com/responsively-org/responsively-app/assets/12045730/fe96c376-ae2d-4c27-9f54-2fb03abb783b)
![her lightness](https://github.com/responsively-org/responsively-app/assets/12045730/8bc6577b-a71c-46b7-b02e-143c3f58aaa5)

**Reply by macx** [reply]
Very low contrast and looked like a truck with two circles below the blocks. But I like the direction.

**Reply by markus-kraemer** [reply]
Now i see the truck, too :)

**Reply by markus-kraemer** [reply]
![Extra Large 02@1x](https://github.com/responsively-org/responsively-app/assets/12045730/68b94f83-7ea3-4cea-b7cf-0123511990a3)

**Reply by markus-kraemer** [reply]
![Uploading Artboard.png…]()

**Reply by markus-kraemer** [reply]
No borders and more Contrast on the Letters. Looks bolder 😁
